demos/linux_rpc.rst

```diff
@@ -0,0 +1,94 @@
+
+======================
+OpenAMP Linux RPC Demo
+======================
+
+.. _linux-rpc-intro:
+
+***************
+Linux RPC Intro
+***************
+
+The RPMsg Multi Services reference sample demonstrates OpenAMP :ref:`Remote Procedure Call (RPC)<overview-proxy-rpc-work-label>` components between two linux processors, one as the host and the other as remote.
+
+..  image::  ../images/demos/linux-rpc-intro.svg
+
+
+.. _linux-rpc-components:
+
+********************
+Linux RPC Components
+********************
+
+This demonstration shows how the :ref:`Remote Procedure Call (RPC)<overview-proxy-rpc-work-label>` component is a shim over RPMSg providing the familiar Input/Output (IO) commands like open, read, write, close both for files and standard input/output.
+
+The underlying OpenAMP architectural components used by these applications are
+
+* :ref:`Remoteproc<overview-remoteproc-work-label>`
+* :ref:`Resource Table<overview-remoteproc-work-label>`
+* :ref:`RPMsg<overview-rpmsg-work-label>`
+* :ref:`Virtio<overview-rpmsg-work-label>`
+* :ref:`RPC<overview-proxy-rpc-work-label>`
+* :ref:`Libmetal<overview-proxy-libmetal-label>`
+
+The Linux file and stdio access is performed in the `linux_rpc_demod application <https://github.com/OpenAMP/open-amp/blob/main/apps/examples/linux_rpc_demo/linux_rpc_demod.c>`_.
+
+The following architecture diagram shows the primary components involved in the demonstration.
+
+..  image::  ../images/demos/linux-rpc-components.svg
+
+.. _linux-rpc-control-flow:
+
+The top-level control flow is shown in the following message diagram.
```

Comment:
It's not clear if the server actions from "Setup Resource Table" to "Create RPMsg End Point" are meant to show what happens in platform_init (1) or if they are superfluous and they should be removed (2).

If (1), then any similar control flow diagrams would need to be updated to match this one (e.g. ../images/demos/echo-test-control-flow.svg).

If (2), please remove them from the Server timeline.